poetry.lock

```diff
@@ -3618,32 +3525,20 @@ files = [
 
 [[package]]
 name = "textfsm"
-version = "1.1.3"
+version = "1.1.2"
```

Comment:
Worth figuring out why the hard max of `textfsm` coming in from `napalm`. Guessing because `netmiko` pins it?